sources/models/shared-defaults/docker-services.toml

```diff
@@ -6,6 +6,6 @@ configuration-files = ["docker-daemon-config", "proxy-env"]
 path = "/etc/docker/daemon.json"
 template-path = "/usr/share/templates/docker-daemon-json"
 
-# Image registries
-[metadata.settings.container-registry]
+# Image registry mirrors
+[metadata.settings.container-registry.mirrors]
```

Comment:
Is there a model change? Or are you making the restart condition more specific?

Reply:
I'm making the restart condition more specific. `docker` is only impacted by `container-registry.mirrors` and not `container-registry.credentials`